### PR TITLE
Add states and rename CargoHandler

### PR DESCRIPTION
**Pull request by j-james**
Removed conflicting files

### CODE REVIEW

File: src/main/java/com/spartronics4915/frc2019/subsystems/ArmIntake.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    }
+    private final ILoop mLoop = new ILoop()
+    {
+
+        @Override
+        public void onStart(double timestamp)
+        {
+            synchronized (ArmIntake.this)
+            {
+                mWantedState = WantedState.START_WANTEDSTATE; // XXX: What should the starting WantedState and SystemState be?
```
Comment (ronan33) [inline src/main/java/com/spartronics4915/frc2019/subsystems/ArmIntake.java]: add these initial states

File: src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java [hunk 2]
```diff
@@ -4,36 +4,45 @@
...
+            mInstance = new CargoChute();
         }
         return mInstance;
     }
 
     public enum WantedState
     {
-        CLOSED, INTAKE,
+        /* Each WantedState corresponds to a button on the robot
+         * RAMP_INTAKE - button is a manual override. Should activate when a sensor detects a ball intaked.
```
Comment (ronan33) [inline src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java]: this doesn't line up with an actual state

File: src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java [hunk 3]
```diff
@@ -55,25 +64,28 @@ private CargoHandler()
         @Override
         public void onStart(double timestamp)
         {
-            synchronized (CargoHandler.this)
+            synchronized (CargoChute.this)
             {
-                mWantedState = WantedState.CLOSED;
+                mWantedState = WantedState.CLOSED; // XXX: What should the starting states be?
```
Comment (ronan33) [inline src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java]: starting states

File: src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+    public static CargoChute getInstance()
+    {
+        if (mInstance == null)
+        {
+            mInstance = new CargoChute();
+        }
+        return mInstance;
+    }
+
+    public enum WantedState // Each WantedState will correspond to a button
```
Comment (pietroglyph) [inline src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java]: What `WantedState` do we use if we just want the chute to hold the cargo at the top?

**Comment by pietroglyph** [comment]
I'm going to merge this, but it needs more work which we'll deal with later.

**Pull request merged**